src/creditCardList.js

```diff
@@ -20,6 +20,7 @@ const CREDIT_CARD_LIST = [
   }, {
     name: 'Mastercard',
     regexpFull: '^5[1-5][0-9]{14}$'
+    //regexpFull: '^((5[1-5][0-9]{14})|(2221[0-9]{12})|(222[2-9][0-9]{12})|(22[3-9][0-9]{13})|(2[3-6][0-9]{14})|(27[01][0-9]{13})|(2720[0-9]{12}))$' //After October
```

Comment:
In the PR description is mentioned `2222`, but in the regex it turned into `(222[2-9][0-9]{12})`, which is not the same.

In the PR description is mentioned `223`, but in the regex it turned into `(22[3-9][0-9]{13})`, which is not the same.

Also, the inner parenthesis are not necessary:

```
From: ^((5[1-5][0-9]{14})|(2221[0-9]{12})|(222[2-9][0-9]{12})|(22[3-9][0-9]{13})|(2[3-6][0-9]{14})|(27[01][0-9]{13})|(2720[0-9]{12}))$
To  : ^(5[1-5][0-9]{14}|2221[0-9]{12}|222[2-9][0-9]{12}|22[3-9][0-9]{13}|2[3-6][0-9]{14}|27[01][0-9]{13}|2720[0-9]{12})$
```